server/src/routes/api/v1/pizzaPlaceRouter.js

```diff
@@ -40,4 +41,6 @@ pizzaPlaceRouter.get("/:id", async (req, res) => {
 	}
 })
 
+pizzaPlaceRouter.use("/:id/reviews/new", pizzaPlaceReviewRouter)
```

Comment:
this should be `/:id/reviews`, not including /new

When posting to create a new review the proper path is `POST /pizza-places/:id/reviews`